ios/browser/api/features/features.mm

```diff
@@ -339,4 +340,9 @@ + (Feature*)kBraveAppleTranslateEnabled {
       initWithFeature:&brave::features::kBraveAppleTranslateEnabled];
 }
 
++ (Feature*)kUseBraveUserAgent {
```

Comment:
@kylehickinson we need a action target or something to automatically translate features to ios similar to what java does